### PR TITLE
Fabo/982 fix login not working

### LINKED ISSUE #982: Password input fails in the beginning
UI Version: 0.7.1

Description:

When I first enter my password it fails. Maybe the binary has not started yet?

### PR DESCRIPTION
**Pull request by faboweb**
Closes #982 

Description:

This disabled the signin if not connected to a node. I hope this fixes the problem.

❤️ Thank you!

### CODE REVIEW

File: app/src/renderer/components/common/TmSessionImport.vue [hunk 1]
```diff
@@ -73,6 +74,9 @@ export default {
     TmFormMsg,
     TmFormStruct
   },
+  computed: {
+    ...mapGetters(["connected"])
```
Comment (NodeGuy) [inline app/src/renderer/components/common/TmSessionImport.vue]: I think we should stick to using only standard features of JavaScript.  We want our code to be used by other developers.

File: app/src/renderer/components/common/TmSessionSignIn.vue [hunk 2]
```diff
@@ -52,7 +53,7 @@ export default {
     }
   }),
   computed: {
-    ...mapGetters(["user", "mockedConnector", "lastHeader"]),
+    ...mapGetters(["user", "mockedConnector", "lastHeader", "connected"]),
```
Comment (NodeGuy) [inline app/src/renderer/components/common/TmSessionSignIn.vue]: same as above

File: app/src/renderer/components/common/TmSessionSignUp.vue [hunk 3]
```diff
@@ -77,6 +79,9 @@ export default {
     TmFormMsg,
     TmFormStruct
   },
+  computed: {
+    ...mapGetters(["connected"])
```
Comment (NodeGuy) [inline app/src/renderer/components/common/TmSessionSignUp.vue]: same as above

File: app/src/renderer/components/common/TmSessionImport.vue (hunk 1 above)
Comment (faboweb) [reply app/src/renderer/components/common/TmSessionImport.vue]: ?? This is a feature of vuex. Could you elaborate what you mean and what you propose as an alternative?

Comment (NodeGuy) [reply app/src/renderer/components/common/TmSessionImport.vue]: I mean the [object spread property is only a proposal](https://github.com/tc39/proposal-object-rest-spread) and is not part of JavaScript.

As an alternative you can write the following in JavaScript:

```JavaScript
computed: mapGetters(["connected"]),
```

Comment (faboweb) [reply app/src/renderer/components/common/TmSessionImport.vue]: We would need to change this in the whole application, as it is used like this in almost every component. I think this change doesn't belong into this PR.
Let's discuss this in the next meeting, ok?

Comment (NodeGuy) [reply app/src/renderer/components/common/TmSessionImport.vue]: I don't understand why it needs to be changed all at once but I'm happy to discuss in the meeting if you like.

**Pull request merged**